learn/user-guide/running-ballerina-code.md

```diff
@@ -92,9 +93,11 @@ $ bal run main.jar
 
 ### Ballerina Runtime Configurable Variables
 
-A Ballerina runtime can be configured using configurable variables, which can be configured using `Config.toml` file. 
-`Config.toml` file may or may not provide configuration values for configurable variables that are used in the program. 
-Ballerina currently supports configurable variables of types `int`, `float`, `boolean`, `string`, `decimal` and the arrays of respective types. 
+A Ballerina runtime can be configured using configurable variables.
+The values for `configurable` variables can be provided through CLI parameters and configuration files with
+loading and override precedence in the same order.
```

Comment:
Shall we break this into two sentences? Lats part we can say 
When loading the values to configurable variables, command-line args get the higher precedence. Then configuration TOML files based on how the files are located.